include/seqan3/range/container/bitcompressed_vector.hpp

```diff
@@ -518,6 +518,19 @@ class bitcompressed_vector
         return (*this)[size()-1];
     }
 
+    //!\brief Direct access to the underlying SDSL vector.
```

Comment:
Maybe you should describe more clearly what the type will be as this is exposed to the API. Note that if we expose this to the API, we cannot change the internal type anymore without breakage.

Reply:
I changed it a bit. I'd like to link to a documentation of the bitvector. But there are no online docs for the sdsl3 bitvector. And the sdsl2 one looks like [this](https://algo2.iti.kit.edu/gog/docs/html/namespacesdsl.html#a93df8f263fced9ae9a8b0f9f37e444eb).

Anyway, I noticed that in `concatenated_sequences` we added this sentence:
`This exact representation of the data is implementation defined. Do not rely on it for API stability.`
So it's basically not API and we could add `\noapi` for doxygen to this function?

Then we could discuss if all our container `data`/`raw_data` functions are no api.

Reply:
> So it's basically not API and we could add \noapi for doxygen to this function?

:+1:

> Then we could discuss if all our container data/raw_data functions are no api.

:+1: